olive/extra_dependencies.json

```diff
@@ -24,5 +24,8 @@
     ],
     "inc": [
         "neural-compressor"
+    ],
+    "optimum": [
```

Comment:
Please also add the pass dependencies here https://github.com/microsoft/Olive/blob/main/olive/workflows/run/run.py#L64